DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -341,6 +339,21 @@
     <![CDATA[delete track_.exchange(nullptr);
     ]]>
   </ioread>
+  <ioread sourceClass="pat::PackedCandidate" source="uint16_t packedCovarianceDxyDxy_; uint16_t packedCovarianceDxyDz_; uint16_t packedCovarianceDzDz_;   int8_t packedCovarianceDlambdaDz_; int8_t packedCovarianceDphiDxy_;  int8_t packedCovarianceDptDpt_; int8_t packedCovarianceDetaDeta_;  int8_t packedCovarianceDphiDphi_ " version="[-27]" targetClass="pat::PackedCandidate"
```

Comment:
???  look, backward compatibility for miniaod is not granted, so IF we manage to get it working in a backward compatible way I can certainly try to target all releases for which we actually produced some miniaod (this I meant when I said "went  into production"). pre2 wasn't (and will not be) used for any massive miniaod production. Anyhow, I said I will update to 28 once I get a working version of that... the current want is anyhow not yet working... so why do you care?